scripts/docker/Dockerfile.openmptarget

```diff
@@ -38,7 +38,7 @@ RUN CMAKE_URL=https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSIO
     rm ${CMAKE_SCRIPT}
 ENV PATH=${CMAKE_DIR}/bin:$PATH
 
-ARG LLVM_VERSION=llvmorg-14.0.0
```

Comment:
Is this necessary for this pull request?

Reply:
Not necessary for this. I reverted back to 14. Will update that in a different PR.